index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Setup and Mibile firs</title>
+</head>
+<body>
+        <div class="header">
+            <div class="header-container">
+                <div class="status-bar">
+                   <ul class="topbar">
+                    <li><img src="images/time.png" alt=""></li>
+                    <li><img src="images/network.png" alt=""></li>
+                   </ul>
+                </div>
```

Comment:
- [ ] [OPTIONAL] Even though this appears on the Figma design, I would recommend you remove it since you will use this portfolio in the future and  I suppose you won't want this to appear as part of your portfolio. ❓ ❔ ❓ ❔